packages/react/src/components/TreeView/TreeNode.tsx

```diff
@@ -203,13 +373,19 @@ const TreeNode = React.forwardRef(
     }
     return (
       // eslint-disable-next-line jsx-a11y/role-supports-aria-props
-      <li {...treeNodeProps} aria-expanded={!!expanded} ref={ref}>
+      <li
+        {...treeNodeProps}
+        aria-expanded={!!expanded}
+        ref={ref}
+        aria-current={treeNodeProps['aria-current'] || undefined}
+        aria-selected={treeNodeProps['aria-selected'] || undefined}
+        id={treeNodeProps.id as string}>
         <div className={`${prefix}--tree-node__label`} ref={currentNodeLabel}>
           {/* https://github.com/carbon-design-system/carbon/pull/6008#issuecomment-675738670 */}
           {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
           <span
             className={`${prefix}--tree-parent-node__toggle`}
-            disabled={disabled}
+            // disabled={disabled}
```

Comment:
I think this needs to remain

Reply:
If I add it back I get this error
![image](https://github.com/carbon-design-system/carbon/assets/25573926/96b9d8ae-7d37-4228-bd45-86bb93bd8c23)

Basically what it is trying to say is that span element doesn't have a `disabled` prop. And it makes sense because only html elements such as button, input, select, textarea have the `disabled` attribute

Reply:
Oh my bad, yeah let's just remove it then